app/src/utils/apiHelpers.test.tsx

```diff
@@ -0,0 +1,212 @@
+import {
+  getDivisionData,
+  getFeaturedImage,
+  getFeaturedItemData,
+  getHomePageData,
+  getItemsCountFromUUIDs,
+  getNumDigitizedItems,
+  getRandomFeaturedItem,
+} from "./apiHelpers";
+import { apiResponse } from "./apiResponse";
+import defaultFeaturedItem from "../data/defaultFeaturedItemData";
+import {
+  mockFeaturedItemResponse,
+  mockItemResponse,
+} from "__tests__/__mocks__/data/mockApiResponses";
+
+jest.mock("./apiResponse");
+
+beforeEach(() => {
+  jest.clearAllMocks();
+});
+
+describe("getHomePageData", () => {
+  it("creates response containing random number and all 7 lanes", async () => {
+    (apiResponse as jest.Mock).mockResolvedValueOnce(
+      Promise.resolve({
+        nyplAPI: {
+          response: {
+            counts: {
+              count: [
+                {
+                  uuid: { $: "test" },
+                  count_value: { $: "10" },
+                },
+              ],
+            },
+          },
+        },
+      })
+    );
+    const result = await getHomePageData();
+    expect(apiResponse as jest.Mock).toHaveBeenCalled();
+    expect([0, 1]).toContain(result.randomNumber);
+    expect(result.lanesWithNumItems.length).toEqual(7);
+    // Test uuid response doesn't match the collection uuid, so all of these should be 0.
+    expect(
+      result.lanesWithNumItems[0].collections[3].numberOfDigitizedItems
+    ).toEqual("0");
+  });
+});
+
+describe("getFeaturedItemData", () => {
+  it("creates response containing featuredItem and numDigitizedItems", async () => {
```

Comment:
This is testing the default values when `getFeaturedImage` and `getNumDigitizedItems` don't return data from their own API call. I think it's fair to only test this function through the default response and test the other two functions separately (so you don't need an integration test). Just make a note in the description that it's returning default fallback data.